cruds_adminlte/templates/HttpResponse/403.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```

Comment:
Try to extend from base.html, find some html on templates/cruds/ and look how to extends from base file.